src/utils/FormLabel.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface FormLabelProps{
+    labelName:string,
+    id:string,
+    classes:string
```

Comment:
You can rename this to className, so that tailwind intelliscense can recognize this and its more reeadable as well